tests/e2e/baseTest.ts

```diff
@@ -83,12 +89,15 @@ interface VSCodeFixtures {
 }
 
 export const test = testBase.extend<VSCodeFixtures>({
-  electronApp: async ({ trace }, use, testInfo) => {
-    const testConfigs = getTestSetupCache();
-
-    // create a temporary directory for this test run
+  testTempDir: async ({}, use) => {
```

Comment:
Setting this up as a standalone fixture allows any other fixtures (or the currently-running test) to read/write files to be bundled with the test report, and is required for saving extension+sidecar logs.